Game/Resources/elevatorScene/Scripts/elevator_frontDoorCon.as

```diff
@@ -64,17 +72,21 @@ class elevator_frontDoorCon {
 
                 doorWithX.SetWorldPosition(tempPosWithX);
                 doorWithService.SetWorldPosition(tempPosWithService);
+                timer += deltaTime;
+                doorOpen = !(timer >= doorTime);
             }
         }
     }
 
     void OpenDoor() {
         openDoor = true;
+        timer = 0;
         doorWithX.GetSoundSource().Play();
         doorWithService.GetSoundSource().Play();
     }
 
     void CloseDoor() {
+        timer = 0;
```

Comment:
Minimal thing, as I don't know if the compiler converts the the int to a float at compiletime or runtime. It's probably better to explicit set the timer to a float. (timer = 0.0f)